openqa-trigger-bisect-jobs

```diff
@@ -111,7 +111,8 @@ client_args = [
 def call(cmds, dry_run=False):
     log.debug("call: %s" % cmds)
     return subprocess.check_output(
-        (["echo", "Simulating: "] if dry_run else []) + cmds
+        (["echo", "Simulating: "] if dry_run else []) + cmds,
+        stderr=subprocess.STDOUT
```

Comment:
Why change this? I don't seev why that should be needed from the git commit messages

Reply:
u are right. I should be able to write a comment about it but I dont have a good reasoning. During testing this and that I found that this would ensure the output in the console. It worked, it stayed. Later on, ofc things changed.

Reply:
But the code have adjusted based on others' with this "accepted". it is not a problem. maybe just a different approach